### PR TITLE
License, jsonpFunction, and tests

### PR DESCRIPTION
**Pull request by joeldenning**
See https://github.com/openmrs/openmrs-esm-login/pull/30

### CODE REVIEW

File: __mocks__/dimensions.mock.ts [hunk 1]
```diff
@@ -1,3 +1,7 @@
+import dayjs from "dayjs";
+
+const todaysDate = dayjs().format("YYYY-MM-DD");
```
Comment (joeldenning) [inline __mocks__/dimensions.mock.ts]: Some of the tests were dependent on the date being november 13, 2019. I changed that.

File: __mocks__/dimensions.mock.ts [hunk 2]
```diff
@@ -185,8 +189,8 @@ export const mockDimensionResponse = {
           context: {
             reference: "Encounter/1a82fdc6-3d14-44c4-ab45-d65baa0062ae"
           },
-          effectiveDateTime: "2016-12-18T06:48:20+00:00",
```
Comment (joeldenning) [inline __mocks__/dimensions.mock.ts]: The `+00:00` asserts time zone in a way that can actually change which day it is for your local computer. That's why I removed it

File: src/summary/documentation/dimensions-card-level-two.test.tsx [hunk 3]
```diff
@@ -71,7 +71,9 @@ it("renders dates according to designs", () => {
   );
   expect(formatDate(sometimeThisYear)).toBe(`26-Apr 06:49 AM`);
   expect(formatDate(today.toString())).toBe(
-    `Today ${get12Hour(today.getHours())}:${zeroBase(today.getMinutes())} PM`
+    `Today ${get12Hour(today.getHours())}:${zeroBase(today.getMinutes())} ${
+      today.getHours() < 12 ? "A" : "P"
```
Comment (joeldenning) [inline src/summary/documentation/dimensions-card-level-two.test.tsx]: This test would only work if run in the morning 😄

Comment (fatmali) [reply src/summary/documentation/dimensions-card-level-two.test.tsx]: `A` or `AM`?

Comment (fatmali) [reply src/summary/documentation/dimensions-card-level-two.test.tsx]: oops saw the `M`

Comment (joeldenning) [reply src/summary/documentation/dimensions-card-level-two.test.tsx]: The `M` is on the next line (line 76) - it's there for both AM and PM which is why it's not in the ternary

**Pull request merged**